### DISCUSSION TITLE
srsLog error - The backend queue size is about to reach its maximum capacity of 8192 elements, new log entries will get discarded. Consider increasing the queue capacity.

### DISCUSSION THREAD

**Discussion by AzeezEbrahim** [srsran/srsRAN_Project#362] [General Help] [answered]
Can any explain what is this and why it happens?
```
srsLog error - The backend queue size is about to reach its maximum capacity of 8192 elements, new log entries will get discarded.
Consider increasing the queue capacity.
```

**Comment by ismagom** [comment, marked as answer]
This is because the log level is too high, just reduce it to info or warning

**Reply by frankist** [reply]
Optionally, just enable debug logging in the layers of interest. RLC, MAC and PHY are the more verbose layers.

**Comment by houshangaziziaccelleran** [comment]
Hi

I am experiencing the same with my multi-UE testing (TCP-DL Data) even with log level set to `error`.

`srsLog error - The backend queue size is about to reach its maximum capacity of 8192 elements, new log entries will get discarded.`

I need to highlight that the same multi-UE test with the same log level is passed (no issue if the Data involved in the test is UDP). Any thoughts please?

Logs attached! No `gnb.log` as it was set to `error`.
[tcp-dl-ues-lost.zip](https://github.com/srsran/srsRAN_Project/files/15250552/tcp-dl-ues-lost.zip)

**Comment by houshangaziziaccelleran** [comment]
Hi @ismagom Another set of logs for the same issue. This time it gives segmentation fault.
Test:

- Attach 64 UEs
- Run TCP-DL on all UEs
[tcp-dl-segmentation-fault.zip](https://github.com/srsran/srsRAN_Project/files/15250786/tcp-dl-segmentation-fault.zip)

**Reply by ismagom** [reply]
This is a different issue, can you create a new one please?

Also, explain your setup in detail please

**Comment by houshangaziziaccelleran** [comment]
Thanks @ismagom 
#608 just raised with complete explanation on setup/test/issue.
